TheJumpCloud/support
# CUT-4043: Remove SystemToken from Get-JCSystem

## Issues
* [CUT-4043](https://jumpcloud.atlassian.net/browse/CUT-4043) - Remove SystemToken from Get-JCSystem

## What does this solve?
Removes the SystemToken attribute from the Get-JCSystem returnProperties param as well as the return object

## Is there anything particularly tricky?
N/A

## How should this be tested?
Run Get-JCSystem -returnProperties systemToken - This should error
Run Get-JCSystem | Select -First 1 - Validate that the systemToken property is not present
Run Get-JCSystem -Id <SystemID> - Validate that the systemToken property is not present


[CUT-4043]: https://jumpcloud.atlassian.net/browse/CUT-4043?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
